app/build.gradle

```diff
@@ -38,6 +53,45 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
     ndkVersion '21.3.6528147'
+
+    flavorDimensions = ['layout']
+
```

Comment:
I do not see the point for these flavours.
I could see a specific build flavour for lcevc and one for the default but really there should not be a need for this.
The native libs are pushed when running the tests and compared to the sources that should be very little overhead (and cached).

It is essential that default builds should run without any external dependencies that are not needed the default test cases. Any external dependency should be build and handled outside of gradle.